.gitignore

```diff
@@ -1,3 +1,4 @@
 *.pyc
 .piolibdeps
 .pio
+__pycache__
```

Comment:
Probably missing EOL.

Reply:
Added EOL in https://github.com/platformio/platform-renesas-ra/pull/7/commits/170b200dc9b2bebbeb37b29fad9b4bcb1abe94bd